src/prefect/settings/models/server/root.py

```diff
@@ -111,6 +112,10 @@ class ServerSettings(PrefectBaseSettings):
         default_factory=ServerAPISettings,
         description="Settings for controlling API server behavior",
     )
+    concurrency: ServerConcurrencySettings = Field(
+        default_factory=ServerConcurrencySettings,
+        description="Settings for controlling server-side concurrency limit handling",
+    )
```

Comment:
just a thought and not for this PR, wonder if we can use `ImportString` here with pydantic-settings, im not sure why not

Reply:
I tried that, and it makes the settings unserializable. We might be able to do it with a custom serializer.